api/src/main/scala/org/ensime/api/common.scala

```diff
@@ -91,10 +93,10 @@ object RefactorType {
 }
 
 final case class SourceFileInfo(
-    file: File,
+    path: Path,
```

Comment:
this is going to involve quite a lot of changes to the clients. Maybe keep the field name as `file` to reduce the impact.